frontend/src/app/main/ui/workspace/tokens/modals/themes.cljs

```diff
@@ -158,40 +159,35 @@
        (tr "workspace.token.create-theme-title")]]]))
 
 (mf/defc theme-inputs
-  [{:keys [theme dropdown-open? on-close-dropdown on-toggle-dropdown on-change-field]}]
+  [{:keys [theme on-change-field]}]
   (let [theme-groups (mf/deref refs/workspace-token-theme-groups)
-        group-input-ref (mf/use-ref)
+        theme-name-ref (mf/use-ref (:name theme))
+        options (map (fn [group]
+                       {:label group
+                        :id group})
+                     theme-groups)
         on-update-group (partial on-change-field :group)
-        on-update-name (partial on-change-field :name)]
+        on-update-name
+        (mf/use-fn
+         (fn [event]
+           (let [value (.-value (.-currentTarget event))]
```

Comment:
There is a namespace called dom where we have fns to get -current-target or get-value
```clojure
target  (dom/get-current-target event)
name   (dom/get-value target)
```
we can do this in one line also.

Reply:
Yep, that helps should be used instead of that interop because they provide the right compiler hints for the interop calls that avoid issues with advanced compilation modes where almost all functions are renamed.

Reply:
:ok_hand: done